crates/zkvm/lib/src/utils.rs

```diff
@@ -1,7 +1,10 @@
 pub trait AffinePoint<const N: usize>: Clone + Sized {
     /// The generator.
+    #[deprecated = "This const will have the `Self` type in the next major version!"]
     const GENERATOR: [u32; N];
 
+    const GENERATOR_T: Self;
```

Comment:
Add note that in this next major version, this should be switched to be `GENERATOR`, instead of `GENERATOR_T`.